grafana-plugin/src/containers/PluginConfigPage/PluginConfigPage.tsx

```diff
@@ -221,14 +235,10 @@ export const PluginConfigPage = (props: Props) => {
           )}
           <p>{'Plugin <-> backend connection status'}</p>
           <pre>
-            <Text type="link">{pluginStatusMessage}</Text>
```

Comment:
display the error as text, not a link, since it is not clickable.

imo makes it a bit more readable/less confusing, thoughts?